drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/test/java/org/drools/workbench/services/verifier/plugin/client/fromfile/DecisionTableAnalyzerFromFileTest.java

```diff
@@ -127,6 +129,29 @@ public void testFileLargeFileGDST() throws
                            CheckType.EMPTY_RULE);
     }
 
+    @Test
+    public void emptyValueListOnColumnShouldNotCountAsAnEnum() throws
+            Exception {
+
+        analyzerProvider.getFactTypes().add(new FactTypes.FactType("Data",
+                                                                   new HashSet<FactTypes.Field>() {{
```

Comment:
could we please use https://github.com/kiegroup/kie-soup/blob/master/kie-soup-commons/src/main/java/org/kie/soup/commons/util/Sets.java ? Those builders were created to eliminate double brace static field initialization - it is marked as bad pattern by sonar.

Reply:
Done